Makefile

```diff
@@ -29,7 +29,7 @@ widgets: \
 	www/widgets/lab12-browser.html www/widgets/lab12.js \
 	www/widgets/lab13-browser.html www/widgets/lab13.js
 
-src/lab%.full.py: src/lab%.py
+src/lab%.full.py: src/lab%.py infra/inline.py infra/asttools.py
```

Comment:
Does this change cause `make test` to fail if the behavior regresses in the future?

Reply:
No, just helps work on the inliner. It would be good to auto test the inlined version but I don’t do that here.